plugins/ltac/g_tactic.mlg

```diff
@@ -54,6 +54,14 @@ let test_lpar_idnum_coloneq =
     lk_kw "(" >> (lk_ident <+> lk_nat) >> lk_kw ":="
   end
 
+(* parse [= without forbidding notations
+   with token [ then any token starting with = *)
+let test_lsq_equal =
+  let open Pcoq.Lookahead in
+  to_entry "test_lsq_equal" begin
+    lk_kw "[" >> lk_kw "=" >> check_no_space
+  end
+
```

Comment:
Why not less enigmatically "left_square_bracket_equal", or "leftsquarebracket_equal"?

Reply:
@herbelin Good point, will make the change if we decide for this solution.
There is an ongoing discussion as to whether we should remove the `[=` token (this PR #16788 ) or hack ssreflect to use it (PR #16789 ). Feel free to give your opinion if any.